### PR TITLE
add standard and prepare for standard lint

### PR DESCRIPTION
**Pull request by Uzlopak**
This PR is also for discussing  if we should switch to standardjs, like fastify core.

This PR adds all the changes, which need to be done manually to be conform with standardjs. It is easier to review the changes, which have to be done manually before doing a lint:fix, than doing a lint:fix and then fixing manually the stuff, which standardjs could not handle. 

I excluded deps/encoding as it is kind of a standalone part and messing with it is potentially critical. Also encoding-indexes.js is a huge file, so having it in a "compressed" format makes sense. 

So if we agree on using standardjs like fastify core does, we should merge this, and in a second PR we would just run `npm run lint:fix` and changing to standard for `npm run lint` in package.json.Also dont forget to add standard to extends attribute of eslint-configuration. 

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Uzlopak** [comment]
Maybe we have to downgrade to eslint 7, till https://github.com/standard/eslint-config-standard/pull/193 lands

**Comment by Uzlopak** [comment]
But we could also independently integrate the code changes to be prepared anyway.

File: test/dicer-multipart-extra-trailer.spec.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 const Dicer = require('../deps/dicer/lib/Dicer');
 const assert = require('assert'),
   fs = require('fs'),
+  path = require('path'),
   inspect = require('util').inspect;
 
-const FIXTURES_ROOT = __dirname + '/fixtures/';
+const FIXTURES_ROOT = path.join(__dirname, '/fixtures/');
```
Comment (kibertoad) [inline test/dicer-multipart-extra-trailer.spec.js:7]: slash before fixture is not needed with join

File: .eslintrc.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
-const warning = process.env['CI'] ? 2 : 1;
-
 module.exports = {
   parserOptions: {
     ecmaVersion: 2018,
   },
+  ignorePatterns: [
```
[the remark this answers is not in the document]
Comment (Uzlopak) [reply .eslintrc.js:5]: did not push... done

File: test/dicer-multipart-extra-trailer.spec.js (hunk 1 above)
Comment (Uzlopak) [reply test/dicer-multipart-extra-trailer.spec.js]: done

**Review by kibertoad — APPROVED** [review]

**Comment by Uzlopak** [comment]
@kibertoad I downgraded eslint for now. Just want to make sure: Still holding on to your approval?

**Comment by kibertoad** [comment]
sure, I'm ok with that

**Pull request merged**